microprofile.jdt/org.eclipse.lsp4mp.jdt.core/src/main/java/org/eclipse/lsp4mp/jdt/core/java/hover/PropertiesHoverParticipant.java

```diff
@@ -130,13 +132,17 @@ public Hover collectHover(JavaHoverContext context, IProgressMonitor monitor) th
 			propertyKey = getAnnotationMemberValue(annotation, annotationMemberName);
 			if (propertyKey != null) {
 				ISourceRange r = ((ISourceReference) annotation).getSourceRange();
-				int offset = annotationSource.indexOf(propertyKey);
-				propertyKeyRange = utils.toRange(typeRoot, r.getOffset() + offset, propertyKey.length());
-
-				if (!hoverPosition.equals(propertyKeyRange.getEnd())
-						&& Ranges.containsPosition(propertyKeyRange, hoverPosition)) {
-					found = true;
-					break;
+				Pattern memberPattern = Pattern.compile(".*[^\\w{\\.](" + annotationMemberName + ")[^\\w}\\.].*", Pattern.DOTALL);

```

Comment:
If you have this

`@Sheduled(cron="cron")`, the pattern returns the second cron (inside the string).

I would exclude " instead of {:

`Pattern memberPattern = Pattern.compile(".*[^\\w"\\.](" + annotationMemberName + ")[^\\w"\\.].*", Pattern.DOTALL);`

Reply:
I've added `\"` to the regular expression to catch this.